lib/Bindings/Python/OMModule.cpp

```diff
@@ -55,6 +56,18 @@ struct Object {
     return omEvaluatorObjectValueGetPrimitive(result);
   }
 
+  // Get a list with the names of all the fields in the Object.
+  std::vector<std::string> getFieldNames() {
+    ArrayAttr fieldNames =
+        cast<ArrayAttr>(unwrap(omEvaluatorObjectGetFieldNames(object)));
+
+    std::vector<std::string> slots;
+    for (auto fieldName : fieldNames.getAsRange<StringAttr>())
```

Comment:
Same for `StringAttr`.

Reply:
Oh sorry, didn't realize that, will create a new PR to fix it.

Reply:
@prithayan no worries, it was my bad, and fixed in https://github.com/llvm/circt/commit/80531ebc4f13d4b0f589567a5a0c18aa0bc0b02e